File: cpp/PetShop.cpp
#include <bits/stdc++.h>

using namespace std;

// kelas petshop
class PetShop
{
private:
    // atribut-atribut private
    int id;
    string name;
    int price;
    int stock;

public:
    // konstruktor kosong
    PetShop()
    {
    }

    // konstruktor dengan parameter untuk mengisi atribut
    PetShop(int id, string name, int price, int stock)
    {
        this->id = id;
        this->name = name;
        this->price = price;
        this->stock = stock;
    }

    // getter dan setter (ambil dan set atribut)
    // setter id
    void setId(int id)
    {
        this->id = id;
    }

    // getter id
    int getId()
    {
        return id;
    }

    // setter name
    void setName(string name)
    {
        this->name = name;
    }

    // getter name
    string getName()
    {
        return name;
    }

    // setter price
    void setPrice(int price)
    {
        this->price = price;
    }

    // getter price
    int getPrice()
    {
        return price;
    }

    // setter stock
    void setStock(int stock)
    {
        this->stock = stock;
    }

    // getter stock
    int getStock()
    {
        return stock;
    }

    ~PetShop()
    {
    }
};

File: phyton/PetShop.py
class PetShop:
    # konstruktor
    def __init__(self, id, name, price, stock):
        self.id = id
        self.name = name
        self.price = price
        self.stock = stock
        
    # getter dan setter untuk id
    def setId(self, id):
        self.id = id
    
    def getId(self):
        return self.id
    
    # getter dan setter untuk name
    def setName(self, name):
        self.name = name
    
    def getName(self):
        return self.name
    
    # getter dan setter untuk price
    def setPrice(self, price):
        self.price = price
    
    def getPrice(self):
        return self.price
    
    # getter dan setter untuk stock
    def setStock(self, stock):
        self.stock = stock
    
    def getStock(self):
        return self.stock